dts-config.js

```diff
@@ -0,0 +1,36 @@
+const libraries = {
+    inlinedLibraries: [
+        '@pixi/colord',
+        'earcut',
+        'eventemitter3',
+    ],
+};
+
+const output = {
+    inlineDeclareGlobals: true,
+    exportReferencedTypes: false,
+    umdModuleName: 'PIXI',
```

Comment:
it is possible that instead of this we might need to use explicit global declarations, something like this:

```ts
import * as PIXIModule from './index';

declare global {
	const PIXI: PIXIModule;
}
```